src/renderer/js/redux/reducers/subscriptions.js

```diff
@@ -0,0 +1,67 @@
+// @flow
```

Comment:
I wasn't seeing `Flow` catch state errors when I moved all of the types into `types/subscriptions.js`, but it is working when kept in the reducer file.

Needs more exploring.

Reply:
interesting :/